### PR TITLE
Provide cluster and usernames from k8 config for login

### LINKED ISSUE #671: Provide user names from k8s config when login to the cluster

### PR DESCRIPTION
**Pull request by mohitsuman**
Fix #671 

**Cluster View**
Screenshot 2019-06-22 at 10 24 36 AM

**UserName**
Screenshot 2019-06-22 at 10 24 46 AM

The new UI also mentions the current context of from k8 config.

### CODE REVIEW

**Comment by sudhirverma** [comment]
testPR

**Comment by sudhirverma** [comment]
testPR

**Comment by mohitsuman** [comment]
testPR

File: src/util/kubeUtils.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import { QuickPickItem } from "vscode";
+import { KubeConfig } from '@kubernetes/client-node';
+
+export async function getClusters(): Promise<QuickPickItem[]> {
+    const kubectlConfig = new KubeConfig();
+    kubectlConfig.loadFromDefault();
+    if (!kubectlConfig) {
```
Comment (dgolovin) [inline src/util/kubeUtils.ts]: This check is not required anymore

File: src/util/kubeUtils.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            label: c.name,
+            description: c.name === currentContextServer ? 'Current Context' : ''
+        };
+    });
+}
+
+export async function getUsers(clusterUrl: string): Promise<QuickPickItem[]> {
+    const kubectlConfig = new KubeConfig();
+    kubectlConfig.loadFromDefault();
+    if (!kubectlConfig) {
```
Comment (dgolovin) [inline src/util/kubeUtils.ts]: not required because it always false.

File: src/util/kubeUtils.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        return [];
+    }
+    const currentContextUser = kubectlConfig.currentContext.split('/')[2];
+    return kubectlConfig.users
```
Comment (dgolovin) [inline src/util/kubeUtils.ts]: What if there is not .kube/config? You might need to use getUsers() method instead.

**Pull request merged**